frontend/src/Admin/components/CreateShowForm.js

```diff
@@ -227,16 +227,34 @@ class CreateShowForm extends Component {
                 </Row>
               </Col>
             </Row>
-            <Button
-              type='submit'
-              color='primary'
-              style={{
-                cursor: 'pointer'
-              }}
-              disabled={isSubmitting}
-            >
-              Create
-            </Button>
+            <Row>
```

Comment:
Without the `Row` and `Col` bootstrap, telling the cancel button to float right put it in the middle of the screen.